mirror/lldap/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM docker.io/nitnelave/lldap:latest-debian@sha256:035f23fcf976129c71d5b40ddac497d3cddf0c5162020dd2e09bb0b6868f4a9d
+FROM docker.io/nitnelave/lldap:stable-debian@sha256:035f23fcf976129c71d5b40ddac497d3cddf0c5162020dd2e09bb0b6868f4a9d
```

```suggestion
FROM docker.io/nitnelave/lldap:v0.4.2@sha256:4c27b311671d878edb6dac8a50c8328fde8a68d68b540a44bfc8a11943475e2d
```
versioned tags are prefered.
Is there any feature that does not work on the alpine, is debian preferred for some feature? (default tag is alpine)

---

Didn't see that tag thanks for picking that up I seen a semver PR but assumed they hadn't implemented it. Debian was proposed following discussions with one of the LLDAP maintainers as the app was having chown issues, the debian version was part of the proposed solution along with the commands in the values.yaml.

---

Cool, switch tag and digest to debian semver  then!!